test/test-get-passwd.c

```diff
@@ -64,19 +64,27 @@ TEST_IMPL(get_passwd) {
   ASSERT(pwd.gid >= 0);
 #endif
 
+#ifdef _WIN32
+  ASSERT(pwd.gecos == NULL);
+#else
+  ASSERT(pwd.gecos != NULL);
+#endif
```

Comment:
Of course, I changed the windows implementation without altering the test. Will fix.